spec/dummy/client/app/packs/rescript-components.js

```diff
@@ -0,0 +1,7 @@
+// This file is used to import your compiled rescript files
+// As per the current configuration, all files are compiled to .bs.js
+// and they are generated on the same directory as the .res file
+
+import { make as HelloWorldReScript } from '../components/HelloWorldReScript.bs';
```

Comment:
FYI If you export a component for JS, you can call it `default` and import it directly without this `make as` dance.